internal/staging/util_test.go

```diff
@@ -42,21 +39,21 @@ var _ = Describe("Util", func() {
 
 			tryInferPullRequestGitlabBranch(gitlabConf, MRiid, gitlabClientGetter)
 
-			g.Expect(gitlabConf).To(Equal(before))
+			g.Expect(gitlabConf).ToNot(Equal(before))
+			g.Expect(gitlabConf.Branch).To(Equal(inferredBranch))
 		})
 
-		It("should infer if true and branch empty", func() {
+		It("interBranch=True defaultBranch='test2'", func() {
```

Comment:
Would it be better to add a description here like what will happen if users set this and that?

Reply:
added. Please help review again.